numpy/lib/arraypad.py

```diff
@@ -410,14 +415,12 @@ def _set_wrap_both(padded, axis, width_pair):
 
     if left_pad > 0:
         # Pad with wrapped values on left side
-        # First slice chunk from right side of the non-pad area.
+        # First slice chunk from left side of the non-pad area.
```

Comment:
In the comment "First slice chunk from *left* side of the non-pad area", actually neither left nor right is correct. It is more of "from middle", but you can think of it as tracing from the left side. (The later comment for right pad is the same.)